src/instruments/src/ISIS/PitchScale.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const ExcessivePitchAttitudeArrows: React.FC = () => (
```

Comment:
**suggestion:** This is static, so it should be used as an `.svg` file referenced by an `<image />` tag.